examples/ref-templates/scripts/htlc.js

```diff
@@ -0,0 +1,40 @@
+const { transferMicroAlgos, transferMicroAlgosLsig } = require("algob");
+
+async function transferAlgo(deployer, senderAddr, receiverAddr, amount, lsig, payFlags){
+    try {
+        const details = await transferMicroAlgosLsig(deployer, senderAddr, receiverAddr, amount, lsig, payFlags);
+        console.log(details);
+    } catch (e) {
+        console.error('Transaction Failed', e);
```

Comment:
I think we should print the error here and not the entire response - `e.response.error`

Reply:
ok